packages/in_app_purchase/in_app_purchase_platform_interface/lib/src/in_app_purchase_platform_addition.dart

```diff
@@ -2,18 +2,34 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore: avoid_classes_with_only_static_members
+import 'package:in_app_purchase_platform_interface/in_app_purchase_platform_interface.dart';
+import 'package:plugin_platform_interface/plugin_platform_interface.dart';
+
 /// The interface that platform implementations must implement when they want to
 /// provide platform specific in_app_purchase features.
```

Comment:
Nit: platform-specific

Reply:
done. Also fixed some other appearance in the plugin.